### PR TITLE
VTCCA-2980 Test case fixed

### PR DESCRIPTION
**Pull request by reddy-yattapu**

### CODE REVIEW

File: test/controllers/ViewAssessmentSpec.scala [hunk 1]
```diff
@@ -43,7 +43,8 @@ class ViewAssessmentSpec extends VoaPropertyLinkingSpec with OptionValues {
     mockDvrCaseManagement,
     StubBusinessRatesValuation,
     mockBusinessRatesAuthorisation,
-    stubMessagesControllerComponents()) {
+    stubMessagesControllerComponents(),
+    false) {
```
Comment (PeterPerhac) [inline test/controllers/ViewAssessmentSpec.scala]: somehow this false fixes something, but it's not clear how. you could use named parameters, especially for boolean parameters

**Comment by habeeb01** [comment]
👍

File: test/controllers/ViewAssessmentSpec.scala (hunk 1 above)
Comment (reddy-yattapu) [reply test/controllers/ViewAssessmentSpec.scala]: I have added name to Flag is isExternalValuationEnabled

**Comment by PeterPerhac** [comment]
👍  happy

**Pull request merged**